settings.gradle

```diff
@@ -8,5 +8,5 @@ include ':helios-benchmarks'
 include ':helios-sample'
 include ':helios-optics'
 include ':helios-dsl-syntax-compiler'
-include 'helios-docs'
+include 'docs'
```

Comment:
Could we move this to the top or last position just for esthetics reasons

Reply:
Ha!  Yeah, sure :nerd_face: